README.md

```diff
@@ -172,3 +172,9 @@ Pull requests are always welcome. Please read through our [Contribution](/docs/c
 ## Frequently Asked Questions (FAQ)
 
 You can read our FAQ, located in our `/docs` directory, [here](/docs/faq.md).
+
+## Data Mapper
+
+To update the data mapper, change the commit SHA in package.json to pick up the desired code revision, then run `yarn install`.  Verify that the projects still builds by running `yarn start -- --aot --prod` and adjust the code as necessary based on the data mapper example usage in the [README](https://github.com/redhat-ipaas/datamapper-ui).
```

Comment:
How about adding a `yarn start:prod` to package.json to simplify the start command?